py/selenium/webdriver/remote/switch_to.py

```diff
@@ -141,7 +141,7 @@ def send_handle(h):
         try:
             # Try using it as a handle first.
             send_handle(window_name)
-        except NoSuchWindowException as e:
```

Comment:
why are removing the "as e" ?

Reply:
0-arg raise restores the last handled exception (and traceback)